src/pages/profile.tsx

```diff
@@ -25,7 +29,10 @@ export default function ProfilePage() {
   const address = isEthereumAddress(params.get('address') || '') ? params.get('address') : userAddress
   const isLoggedUserProfile = userAddress === address
 
-  const { delegation, delegationState, scores, isLoadingScores, ownVotingPower } = useVotingPowerInformation(address)
+  const [grants] = useGrantsByUser(address!, true)
```

Comment:
What if there is no user? I wouldn't force the `!` and make the hook handle an undefined case.

Reply:
But we can see other users profile right? We should be able to view those logged out.

Reply:
You are right, fixed it